### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by laythq**

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+asd
```
Comment (JohnForster) [inline app/views/users/index.html.erb]: What is this?

Comment (JohnForster) [inline app/views/users/index.html.erb]: What is this?

File: spec/models/user_signup_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# require 'rails_helper'
+#
+# RSpec.describe User, type: :model do
+#   before(:each) do
+#     @user = User.new(name:'John', email: 'j_forster@hotmail.co.uk',
```
Comment (JohnForster) [inline spec/models/user_signup_spec.rb]: Remove my email address please.

**Comment by archmagos** [comment]
Looks like the POST #create test is failing

**Comment by JohnForster** [comment]
:shipit:

**Pull request merged**